homeworks/anna.vrana_Anna1o1Vrana/8-components-templates/my-app-gallery/src/app/app.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="container">
+  <h1 class="gallery-title">{{appTitle}}</h1>
```

Comment:
```suggestion
  <h1 class="gallery-title">Gallery</h1>
```
**Не потрібно** прокидати динамічно текст який **завжди** статичний

Reply:
done)